prisma/schema.prisma

```diff
@@ -44,10 +44,12 @@ model Cohort {
 }
 
 model Post {
-  id      Int           @id @default(autoincrement())
-  content String
-  userId  Int
-  user    User          @relation(fields: [userId], references: [id])
+  id        Int           @id @default(autoincrement())
+  content   String
+  userId    Int
+  user      User          @relation(fields: [userId], references: [id])
+  createdAt DateTime?     @default(now())
+  updatedAt DateTime?     @updatedAt
```

Comment:
I don't like that these are optional, these should be required not nullable columns.

Something like this 
```
createdAt DateTime      @default(now())
updatedAt DateTime      @default(now()) @updatedAt
```